.github/workflows/INTEGRATION_TEST.yml

```diff
@@ -55,7 +66,7 @@ jobs:
         id: helm-branch
         run: |
           if [ -z "${{ github.event.inputs.helm-branch }}" ]; then
-            helm_branch=$(jq -r ".[\"${{ github.ref }}\"]" .github/workflows/helm-git-refs.json)
+            helm_branch=$(jq -r ".[\"${{ github.event.inputs.release-branch || github.ref_name }}\"]" .github/workflows/helm-git-refs.json)
```

Comment:
Maybe a naive question, but wouldn't this part:
`${{ github.event.inputs.release-branch || github.ref_name }}`
be evaluated to a boolean?

I did a quick search through the [GitHub expressions docs](https://docs.github.com/en/actions/learn-github-actions/expressions#operators) but didn't find an example like this, but maybe I just missed it or I'm looking in the wrong place 😄

Reply:
Thanks for flagging!

This is a shorthand for the ternary operator in the GitHub expressions docs - basically if the `release-branch` variable is not defined, then use `ref_name`.

You can see that expression being evaluated to a string [in my test run](https://github.com/camunda/connectors/actions/runs/9268204933/job/25496334167) under "Determine Helm chart ref to use".

[Here's another example of it being used elsewhere](https://stackoverflow.com/a/78477094).

I'll go ahead and resolve this, but feel free to ask any follow-ups 😄

Reply:
Got it, thanks for clarifying!